integration-tests/client/chainlink_config_builder.go

```diff
@@ -0,0 +1,151 @@
+	chainConfig v2.Chain,
+	network *blockchain.EVMNetwork,
+) *ChainlinkConfigBuilder {
+	c.config.EVM = append(c.config.EVM, &v2.EVMConfig{
+		ChainID: utils.NewBigI(network.ChainID),
+		Enabled: Pointer(true),
+		Nodes:   c.networkNodes(network),
+		Chain:   chainConfig,
+	})
+	return c
+}
+
+func (c *ChainlinkConfigBuilder) networkNodes(network *blockchain.EVMNetwork) []*v2.Node {
+	nodes := []*v2.Node{}
+	for nodeId := range network.URLs {
+		wsURL, err := models.ParseURL(network.URLs[nodeId])
+		if err != nil {
+			log.Fatal().Str("URL", network.URLs[nodeId]).Err(err).Msg("Error in URL formatting")
+		}
+		httpURL, err := models.ParseURL(network.HTTPURLs[nodeId])
+		if err != nil {
+			log.Fatal().Str("URL", network.HTTPURLs[nodeId]).Err(err).Msg("Error in URL formatting")
+		}
+		nodes = append(nodes, &v2.Node{
+			Name:    Pointer(fmt.Sprintf("node-%d", nodeId)),
+			WSURL:   wsURL,
+			HTTPURL: httpURL,
+		})
+	}
+	return nodes
+}
+
+// AddP2PNetworkingV1 adds defaults for V1 P2P networking
+func (c *ChainlinkConfigBuilder) AddP2PNetworkingV1() *ChainlinkConfigBuilder {
+	c.config.P2P.V1.Enabled = Pointer(true)
+	c.config.P2P.V1.ListenIP = &net.IPv4zero
+	c.config.P2P.V1.ListenPort = Pointer(uint16(6690))
+	return c
+}
+
+// AddP2PNetworkingV2 adds defaults for V2 P2P networking (also enables LogPoller)
+func (c *ChainlinkConfigBuilder) AddP2PNetworkingV2() *ChainlinkConfigBuilder {
+	c.config.P2P.V2.Enabled = Pointer(true)
+	c.config.P2P.V2.ListenAddresses = &[]string{"0.0.0.0:6690"}
+	c.config.P2P.V2.AnnounceAddresses = &[]string{"0.0.0.0:6690"}
+	return c
+}
+
+// AddKeeperDefaults enables default testing behavior for Keepers
+func (c *ChainlinkConfigBuilder) AddKeeperDefaults() *ChainlinkConfigBuilder {
+	c.config.Keeper.TurnFlagEnabled = Pointer(true)
+	c.config.Keeper.TurnLookBack = Pointer(int64(0))
+	c.config.Keeper.Registry.SyncInterval = models.MustNewDuration(time.Second * 5)
+	c.config.Keeper.Registry.PerformGasOverhead = Pointer(uint32(150_000))
+	return c
+}
+
+// EnableOCR enables OCR functionality
+func (c *ChainlinkConfigBuilder) EnableOCR() *ChainlinkConfigBuilder {
+	c.config.OCR.Enabled = Pointer(true)
+	return c
+}
+
+// EnableOCR2 enables OCR2 functionality
+func (c *ChainlinkConfigBuilder) EnableOCR2() *ChainlinkConfigBuilder {
+	c.config.OCR2.Enabled = Pointer(true)
+	return c
+}
+
+// EnableLogPoller enables LogPoller feature
+func (c *ChainlinkConfigBuilder) EnableLogPoller() *ChainlinkConfigBuilder {
+	c.config.Feature.LogPoller = Pointer(true)
+	return c
+}
+
+// MustTOML marshals the config to a TOML string. Will fail if there is an error.
+func (c *ChainlinkConfigBuilder) MustTOML() string {
+	rawTOML, err := toml.Marshal(c.config)
+	if err != nil {
+		log.Fatal().Err(err).Msg("Error marshalling config TOML")
+	}
+	return fmt.Sprintf("%s", rawTOML)
+}
+
+// RawConfig returns the raw config object so you can modify it directly
+func (c *ChainlinkConfigBuilder) RawConfig() *chainlinkConfig.Config {
```

Comment:
If I want to use the default config first and add some customization over it, how do I set ChainlinkConfigBuilder.config and then use `MustTOML`